conf/README.md

```diff
@@ -351,6 +351,7 @@ higher priority).
 * `upgrade_osd_requires_healthy_pgs` - If set to true OSD upgrade process won't start until PGs are healthy.
 * `workaround_mark_disks_as_ssd` - WORKAROUND: mark disks as SSD (not rotational - `0` in `/sys/block/*d*/queue/rotational`)
 * `node_labels` - Comma-separated labels to be applied to the nodes in the cluster, e.g. 'cluster.ocs.openshift.io/openshift-storage="",node-role.kubernetes.io/infra=""', default - empty string
+* `use_config_file` - If set to true the external-cluster-details-exporter python script will use a config file to setup the external cluster.
```

Comment:
BTW if we want to have this covered by some production job we need to add this configuration value to it.

Please talk to someone like @ebenahar in which combination we need to have it covered and we need to update some production job to support this otherwise we will not test it so we will not cover it.